portfolio/src/main/webapp/script.js

```diff
@@ -76,14 +91,25 @@ function scrollToElement(elementId, duration = 300)
  * Function that loads comments
  */
 function LoadComments() {
-    fetch('/data').then(response => response.json()).then((comments) => {
-        const commentsElement = document.getElementById('comments');
+    // Create static commentsLoaded variable, if not exists
+    if(typeof LoadComments.commentsLoaded == 'undefined') {
```

Comment:
Is it a canonical way of creating a static variable?

Reply:
I think so, this is all I could find to avoid declaring a global variable.
Here: https://stackoverflow.com/questions/1535631/static-variables-in-javascript (second answer)
And here: https://electrictoolbox.com/javascript-static-variables/